### PR TITLE
Add openapi generator

### PR DESCRIPTION
**Pull request by Lambourl**

### CODE REVIEW

File: distro.json [hunk 1]
```diff
@@ -243,5 +271,31 @@
...
+        }
+      }
+    }
+  },
+  {
+    "name": "openapi_generator_script",
+    "platforms": {
+      "all": {
+        "universal": {
+          "url": "http://nxxm.indigenious.io/distro/v.0.0.15/script/script_for_openapi_generator.zip",
```
Comment (daminetreg) [inline distro.json]: This should be stored on github, so that we can version it. Can you change that ?

File: distro.json [hunk 2]
```diff
@@ -243,5 +271,31 @@
         }
       }
     }
+  },
+  {
+    "name": "openapi_generator",
+    "platforms": {
+      "all": {
+        "universal": {
+          "url": "http://nxxm.indigenious.io/distro/v.0.0.15/openapi_generator/openapi-generator.zip",
```
Comment (daminetreg) [inline distro.json]: Which version of openapi-generator is that ? could you encode it in the name ?

File: distro.json [hunk 3]
```diff
@@ -243,5 +271,31 @@
         }
       }
     }
+  },
+  {
+    "name": "openapi_generator",
```
Comment (daminetreg) [inline distro.json]: why an underscore if the zip is a dash.

Comment (daminetreg) [reply distro.json]: I would prefer a more consistent naming

**Review by daminetreg — CHANGES_REQUESTED** [review]

File: distro.json (hunk 2 above)
Comment (Lambourl) [reply distro.json]: If I store the scripts on GitHub I won't be able to have the version number in the zip name. 
It's up to you which one you prefer to store under GitHub or have a zip with the version number.

Comment (daminetreg) [reply distro.json]: We are not speaking of the same package, I meant only the script on GH and the openapi-generator tool on our server.

Comment (Lambourl) [reply distro.json]: oki, It works I do this right after the ftxui

**Review by daminetreg — APPROVED** [review]

**Pull request merged**